pipeline/run_benchmark.sh

```diff
@@ -166,6 +169,9 @@ EOT
    WAIT_FOR_LEADER_LAUNCH=3
    CURL_TIMEOUT=20s
 
+   cmd_file=run_init.sh
```

Comment:
I'd say let's create this file in `${LOGDIR}` instead of `$(pwd)` (which I assume is `pipeline` script directory) in order to avoid clobbering if multiple deployments are in progress at the same time.